bin/ream/src/cli/account_manager.rs

```diff
@@ -0,0 +1,79 @@
+use bip32::Mnemonic;
+use clap::Parser;
+use rand::rngs::OsRng;
+use tracing::warn;
+
+const MIN_CHUNK_SIZE: u64 = 4;
+const MIN_LIFETIME: u64 = 18;
+
+#[derive(Debug, Parser)]
+pub struct AccountManagerConfig {
+    /// Verbosity level
+    #[arg(short, long, default_value_t = 3)]
+    pub verbosity: u8,
+
+    /// Account lifetime in 2 ** lifetime slots
+    #[arg(short, long, default_value_t = 28)]
+    pub lifetime: u64,
+
+    /// Chunk size for messages
+    #[arg(short, long, default_value_t = 5)]
+    pub chunk_size: u64,
+
+    /// Seed phrase for key generation
+    #[arg(short, long)]
+    pub seed_phrase: Option<String>,
+}
+
+impl Default for AccountManagerConfig {
+    fn default() -> Self {
+        Self {
+            verbosity: 3,
+            lifetime: 28,
+            chunk_size: 5,
+            seed_phrase: None,
+        }
+    }
+}
+
+impl AccountManagerConfig {
+    pub fn new() -> Self {
+        Self::parse()
+    }
+
+    pub fn validate(&mut self) -> anyhow::Result<()> {
+        // Validate chunk size
+        if self.chunk_size < MIN_CHUNK_SIZE {
+            warn!(
+                "Chunk size must be at least {}, resetting to {}",
+                MIN_CHUNK_SIZE, MIN_CHUNK_SIZE
+            );
```

Comment:
```suggestion
            warn!("Chunk size must be at least {MIN_CHUNK_SIZE}, resetting to {MIN_CHUNK_SIZE}");
```

we should inline this